packages/mui-base/src/SelectUnstyled/SelectUnstyled.tsx

```diff
@@ -278,7 +278,7 @@ SelectUnstyled.propTypes /* remove-proptypes */ = {
   /**
    * The default selected value. Use when the component is not controlled.
    */
-  defaultValue: PropTypes /* @typescript-to-proptypes-ignore */.any,
+  defaultValue: PropTypes.object,
```

Comment:
I can keep the ignore if you want
The generation now works and is more precise (same for other diffs)